project_template/package.json

```diff
@@ -1,52 +1,81 @@
 {
-  "name": "example",
-  "version": "1.0.0",
   "private": true,
+  "name": "##APP_PACKAGE##",
+  "version": "0.0.1",
   "scripts": {
-    "test": "bin/ci",
-    "test:unit": "APP_ENV=test karma start --single-run",
-    "test:integration": "APP_ENV=test PRE_BUILT=true bundle exec rspec",
-    "test:watch": "APP_ENV=test karma start --watch",
-    "start": "make watch",
-    "build": "make dist"
+    "build": "NODE_ENV=${NODE_ENV:-production} webpack",
+    "start": "LIVERELOAD=${LIVERELOAD:-true} script/dev-server.js",
+    "test": "NODE_ENV=${NODE_ENV:-test} karma start config/karma.js --single-run",
```

Comment:
I noticed that `yarn test` is only running unit tests now. Would it make sense to have `yarn test` delegate to `bin/ci` so all tests are ran (unit, integration, linting etc). We could add `test:unit` task then to run karma tests.

/cc @ruisalgado @matthijsgroen

Reply:
agreed... given the conventions around `yarn/npm test`, makes sense to have it run all the available tests